<launch>
	<arg name="tracker_name_left" default="tracker_right" />
	<arg name="tracker_id_right" default="#2" />
	<!-- <arg name="tracker_bus_id_right" default="002@34"/> -->

<!-- 	<include file="$(find openni_launch)/launch/openni.launch">
		<arg name="camera" value="$(arg tracker_name_left)" />
		<arg name="device_id" value="$(arg tracker_id_right)" />
		<arg name="rgb_camera_info_url"   value="file://$(env ROS_HOME)/camera_info/rgb_$(arg tracker_id_right).yaml" />
		<arg name="depth_camera_info_url" value="file://$(env ROS_HOME)/camera_info/depth_$(arg tracker_id_right).yaml" />
	</include> -->
	
	<node name="XnSensorServerKillerNode" output="screen" pkg="openni_tracker_improved" type="openni_tracker" launch-prefix="bash $(find openni_multi_launcher)/xn_sensor_server_cleanup.sh" />
	
	<node name="tracker_right" pkg="wallframe_tracker" type="wallframe_tracker_node" respawn="true">
		<param name="device_id" type="str" value="$(arg tracker_id_right)" />
		<param name="camera_frame_id" type="str" value="/tracker_right_openni_depth_frame" />
		<param name="tf_prefix" value="tracker_right" />
	</node>

</launch>
